pipelines/library/src/common/VersionInfo.groovy

```diff
@@ -250,6 +256,29 @@ class VersionInfo {
         }
     }
 
+    
+    /**
+     * Form ProductVersion with only 4 number with dot , ie 11.0.9.0 or 11.0.9.101
+     */
+    String formMSIProductVersion() {
```

Comment:
Unless there's precedent on this I don't like `formMSIProductVersion()` as a name as I initially thought it was something that was coming from a "filled in form" e.g. a jenkins job build. Maybe `createMSIProductVersion` or `assembleMSIProductVersion` or even `getMSIProductVersion`?

Reply:
I think it's just copying the above method, formSemver(), "form" as in "make", "put together"

Reply:
So precendent is formSemver()

Reply:
is I just copied the style "form" but I'm happy with "get"

Reply:
i'm happy with the precedence

Reply:
I'd be tempted to replace all of them to remove `form` but I'm ok with it for now since this is a bit of a tactical fix anyway :-)